examples/framework_usage/commands/general.rs

```diff
@@ -133,7 +133,7 @@ pub async fn boop(ctx: Context<'_>) -> Result<(), Error> {
         .await?;
 
         mci.create_interaction_response(ctx.discord(), |ir| {
-            ir.kind(serenity::InteractionResponseType::DeferredUpdateMessage)
+            ir.kind(serenity::interaction::InteractionResponseType::DeferredUpdateMessage)
```

Comment:
The use of `serenity::prelude::*` in `poise`'s prelude includes the deprecated variant from `serenity::interactions` and this leads to:
```
error[E0659]: `InteractionResponseType` is ambiguous
   --> src/modal.rs:64:30
    |
64  |             b.kind(serenity::InteractionResponseType::DeferredUpdateMessage)
    |                              ^^^^^^^^^^^^^^^^^^^^^^^ ambiguous name
    |
    = note: ambiguous because of multiple glob imports of a name in the same module
note: `InteractionResponseType` could refer to the enum imported here
   --> src/lib.rs:268:94
    |
268 |                     application_command::*, autocomplete::*, message_component::*, modal::*, *,
    |                                                                                              ^
    = help: consider adding an explicit import of `InteractionResponseType` to disambiguate
note: `InteractionResponseType` could also refer to the type alias imported here
   --> src/lib.rs:272:13
    |
272 |             prelude::*,
    |             ^^^^^^^^^^
    = help: consider adding an explicit import of `InteractionResponseType` to disambiguate
```

Reply:
Weird; when I remove the `interaction::`, it just errors:

```rust
error[E0433]: failed to resolve: could not find `InteractionResponseType` in `serenity`
   --> examples/framework_usage/commands/general.rs:136:31
    |
136 |             ir.kind(serenity::InteractionResponseType::DeferredUpdateMessage)
    |                               ^^^^^^^^^^^^^^^^^^^^^^^ could not find `InteractionResponseType` in `serenity`
```

Adding `pub use serenity::model::application::interaction::InteractionResponseType;` into serenity_prelude fixes all errors.

Am I doing something wrong?? Any idea why I'm not seeing the error you're seeing (which is making much more sense than mine)